components/media-toolbar/readme.md

```diff
@@ -5,6 +5,7 @@ The MediaToolbar component is build to quickly add a complete media editing expe
 ## Usage
 
 ```js
+const { BlockControls } = wp.blockEditor;
```

Comment:
```suggestion
import { BlockControls } from '@wordpress/block-editor';
```

All the examples we have currently use the import syntax because we rely on the `DependencyExtractionWebpackPlugin` anyways :) 

But this is a great addition 👍